cmd/share-secret/storedata.go

```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+type StoreData struct {
+	Secret     string
+	Unit       string
+	Time       int64
+	Registered *time.Time
+	Expire     *time.Time
+}
+
+func (s *StoreData) GetExpirationDate() *time.Time {
```

Comment:
We should not do this. Instead the moment we save the data, we set the expiration manually. Be it a function with a timer, or something else that automatically goes away when it expires